.jvmopts

```diff
@@ -1 +1,5 @@
 -Dcats.effect.stackTracingMode=full
+-Dfile.encoding=UTF8
+-Xms256M
+-Xmx6G
```

Comment:
I put 6G here but I personally use 12. I just wanted to be sure this value would work for everyone/most people